### DISCUSSION TITLE
VM does not boot

### DISCUSSION THREAD

**Discussion by rezagol99** [hashicorp/packer-plugin-vsphere#642] [Community Help]
#### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

When filing a bug, please include the following headings if possible. Any
example text in this template can be deleted.

#### Overview of the Issue

I am trying to build a VM to run in VMware vSphere using the Packer vsphere-iso plugin. Packer hangs on "Waiting for IP". I see that the VM is powered-on, but nothing is running on the console. Looks like that it can not boot. I have the following versions:
Packer: version 1.9.1
VMware ESXi: version 7.0 U3
vCenter: ESXi compatibility Version 7.0 U2. This was originally set for ESX5.5and I upgraded that to ESXi 7.0 U2

#### Reproduction Steps
Just run the attached packer file on a similar VMware setup. Packer starts and hangs on "Waiting for IP"

### Packer version
1.9.1
From `packer version`

### Simplified Packer Template

If the file is longer than a few dozen lines, please include the URL to the
[gist](https://gist.github.com/) of the log or use the [Github detailed
format](https://gist.github.com/ericclemmons/b146fe5da72ca1f706b2ef72a20ac39d)
instead of posting it directly in the issue.

### Operating system and Environment details
The VM is the Windows Server 2019 built on top of ESXi 7.0 U3

OS, Architecture, and any other information you can provide about the
environment.

### Log Fragments and crash.log files
Include appropriate log fragments. If the log is longer than a few dozen lines,
please include the URL to the [gist](https://gist.github.com/) of the log or
use the [Github detailed format](https://gist.github.com/ericclemmons/b146fe5da72ca1f706b2ef72a20ac39d) instead of posting it directly in the issue.

Set the env var `PACKER_LOG=1` for maximum log detail.
[Packer-output.txt](https://github.com/hashicorp/packer/files/12503165/Packer-output.txt)
[MyFiles.zip](https://github.com/hashicorp/packer/files/12502749/MyFiles.zip)

**Comment by tenthirtyam** [comment]
Please transfer to [hashicorp/packer-plugin-vsphere](https://github.com/hashicorp/packer-plugin-vsphere).

cc @nywilken

**Comment by tenthirtyam** [comment]
@rezagol99 - a couple questions:

- Are you statically setting the IP or using DHCP?
- Do you see an address reported in the vSphere Client?
- Does your Packer host have the ability to the network of the guest using SSH (or WinRM for Windows)?
- Do you have more than one network adapter assigned to the build configuration?

**Comment by tenthirtyam** [comment]
This issue has not received any update and will automatically be closed in 7 days if it does not receive any activity by then.

If you find the [community forum](https://discuss.hashicorp.com/c/packer) to be more helpful or if you've found the answer to your question elsewhere please feel free to post a response and close the issue.
